.gitignore

```diff
@@ -20,3 +20,27 @@ qemu/*.efi
 
 # c object
 *.o
+
```

Comment:
Instead of merging this to the zos .gitignore, you can have multiple .gitignore in the sub directories.

So please move this (part) of the gitignore to tools/zos-update-worker and change the paths to be 
`*.exe` instead of `tools/zos-update-worker/*.exe`, etc..